contents/customers/researchgate.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 'How ResearchGate tests product changes for over 25M users'
+customer: ResearchGate
+logo: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/researchgate_logo_1627258295.png
+logoDark: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/resaerchgate_logo_dark_e8ea7ae745.png
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/researchgate_posthog_7a5a2cf3ac.png
+featuredCustomer: false
+industries:
+  - Science
+  - Social network
+users:
+  - Growth
+  - Engineering
+  - Product
+  - Marketing
+toolsUsed:
+  - Experiments
+  - Feature Flags
+  - PostHog Cloud
+  - Insights
+date: 2024-27-06
+---
+
+ResearchGate, the world’s largest professional network for scientists, helps millions of professionals discover and disseminate groundbreaking research annually. The company is driven by a mission to promote access to science — but, with over 160 million publications, the scale of the community starts to present significant challenges.
+
+“We grew up in a time when there weren’t a lot of third party services to solve the sorts of problems that arise from having so many publications and users,” says Paul McCloud, ResearchGate’s Head of Product Engineering. “Thankfully we’ve built a strong engineering team over the last 15 years and we’ve been able to build what we needed in-house.”
+
+Over the years ResearchGate’s team has built everything from internal experimentation frameworks and feature flagging systems to traffic experimentation systems and GraphQL clients. Many of these tools functioned to a high standard, but eventually the maintenance burden became too much. 
+
+“We built these tools well, but there comes a time when we can’t iterate of them as fast and we need to focus elsewhere,” says Paul. “This eventually became a strategic focus: How are we going to migrate more of our stack to third-party services so we can focus on our core competencies?”
```

Comment:
"but there comes a time when we can't iterate **on** them as fast and we need to focus elsewhere"

Reply:
```suggestion
“We built these tools well, but there comes a time when we can’t iterate on them as fast and we need to focus elsewhere,” says Paul. “This eventually became a strategic focus: How are we going to migrate more of our stack to third-party services so we can focus on our core competencies?”
```